docs/data/base/components/slider/slider.md

```diff
@@ -1,193 +1,214 @@
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-:::info
-Hooks give you the most room for customization, but require more work to implement.
-With hooks, you can take full control over how your component is rendered, and define all the custom props and CSS classes you need.
+### Step
 
-You may not need to use hooks unless you find that you're limited by the customization options of their component counterparts—for instance, if your component requires significantly different [structure](#anatomy).
-:::
+The `step` prop snaps the each value to multiples of the given number. In the below example, the input value snaps to increments of `4` starting from the initial `defaultValue`: `3`, `7`, `11`, `15`, and so on.
 
-## Customization
+```tsx
+<Slider.Root step={4} defaultValue={3}>
+  <Slider.Output />
+  <Slider.Track>
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-### Vertical
+You can specify the `largeStep` prop to change the step when the user holds the <kbd>shift</kbd> key, snapping to multiples of 10 by default.
 
-Slider components can be arranged vertically as well as horizontally.
+## Range Sliders
 
-When vertical, you must set `orientation="vertical"` on the Slider so the user can navigate with the up and down arrow keys (rather than the default left-to-right behavior for horizontal sliders).
+To let users set the start and end of a range on a Slider, provide an array of values to the `value` or `defaultValue` prop, and place a `<Slider.Thumb />` for each value in the array:
 
-{{"demo": "VerticalSlider.js"}}
+```tsx
+<Slider.Root defaultValue={[45, 70]}>
+  <Slider.Output />
+  <Slider.Track>
+    <Slider.Thumb />
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-### Discrete sliders
+{{"demo": "RangeSlider.js"}}
 
-The most basic Slider is _continuous_, which means it does not have pre-defined (_discrete_) values for the user to select from.
-This is suitable for situations in which an approximate value is good enough for the user, such as brightness or volume.
+### Overlapping values
 
-But if your users need more precise options, you can create a discrete Slider that snaps the thumb to pre-defined stops along the bar.
-Make sure to adjust the `shiftStep` prop (the granularity with which the slider can step when using Page Up/Down or Shift + Arrow Up/Down) to a value divadable with the `step`.
+The `minDifferenceBetweenValues` prop can be used to to set the mininum difference between values in a range slider, so thumbs do not overlap in the same position. In the below example, the thumbs cannot be moved further towards each other as the difference between values would exceed `minDifferenceBetweenValues`:
 
-To generate a mark for each stop, use `marks={true}`:
+```tsx
+<Slider.Root minDifferenceBetweenValues={5} defaultValue={[15, 20]}>
+  <Slider.Track>
+    <Slider.Thumb />
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-{{"demo": "DiscreteSlider.js"}}
+## Vertical
```

Comment:
What does the `orientation` prop do? I see that changing the orientation doesn't change the behavior of arrow keys

Reply:
It sets `aria-orientation`, and [internally](https://github.com/mui/base-ui/pull/373/files#diff-34aefc046c62ad0585b7d5cd6c4495ebe08f4fa0c306beed4182100aa9d1d667R328-R332) it determines whether to track changes along the x or y axis

It's actually what I tried to explain in the second sentence of this 2-sentence section 😅 I wonder if we need to explain what it does at all

Reply:
side note: arrow key behavior does not change with the orientation https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-temperature/